src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(-n + 1) {
+    background-image: $star-img;
```

Comment:
It seems you're using a SCSS variable `$star-img` for the `background-image`. Ensure that this variable is defined in your SCSS files, typically within `src/styles/utils/variables.scss`. If it's not defined, it will cause a compilation error.